yt_dlp/extractor/gopro.py

```diff
@@ -57,8 +57,8 @@ def _real_extract(self, url):
         video_id = self._match_id(url)
         webpage = self._download_webpage(url, video_id)
 
-        metadata = self._parse_json(
-            self._html_search_regex(r'window\.__reflectData\s*=\s*([^;]+)', webpage, 'metadata'), video_id)
+        metadata = self._search_json(
+            r'<script>window\.__reflectData\s*=', webpage, 'metadata', video_id)
```

Comment:
is adding the `<script>` tag necessary?

I think we should at least do this
```suggestion
            r'<script>\s*window\.__reflectData\s*=', webpage, 'metadata', video_id)
```

Reply:
```suggestion
            r'window\.__reflectData\s*=', webpage, 'metadata', video_id)
```